### ISSUE TITLE
Cannot load such file -- accessibility/extras

### ISSUE THREAD

**Issue by mhenrixon** [RoboticCheese/mac-app-store-chef#26]
My chef run fails with the following:

```    ================================================================================
    Error executing action `open` on resource 'mac_app_store[default]'
    ================================================================================
    
    LoadError
    ---------
    cannot load such file -- accessibility/extras
    
    Cookbook Trace:
    ---------------
    /Users/mhenrixon/Code/github/soloist/sprout-wrap/cookbooks/mac-app-store/libraries/helpers.rb:378:in `require'
    /Users/mhenrixon/Code/github/soloist/sprout-wrap/cookbooks/mac-app-store/libraries/helpers.rb:378:in `current_application_name'
    /Users/mhenrixon/Code/github/soloist/sprout-wrap/cookbooks/mac-app-store/libraries/provider_mac_app_store.rb:126:in `trust_app'
    /Users/mhenrixon/Code/github/soloist/sprout-wrap/cookbooks/mac-app-store/libraries/provider_mac_app_store.rb:116:in `prep'
    /Users/mhenrixon/Code/github/soloist/sprout-wrap/cookbooks/mac-app-store/libraries/provider_mac_app_store.rb:69:in `block in <class:MacAppStore>'
    
    Resource Declaration:
    ---------------------
    # In /Users/mhenrixon/Code/github/soloist/sprout-wrap/cookbooks/mac-app-store/recipes/default.rb
    
     35: mac_app_store 'default' do
     36:   username node['mac_app_store']['username']
     37:   password node['mac_app_store']['password']
     38:   action :open
     39:   notifies :quit, 'mac_app_store[default]'
     40: end
     41: 
    
    Compiled Resource:
    ------------------
    # Declared in /Users/mhenrixon/Code/github/soloist/sprout-wrap/cookbooks/mac-app-store/recipes/default.rb:35:in `from_file'
    
    mac_app_store("default") do
      action [:open]
      updated true
      updated_by_last_action true
      retries 0
      retry_delay 2
      default_guard_interpreter :default
      declared_type :mac_app_store
      cookbook_name :"mac-app-store"
      recipe_name "default"
      username "ENV[\"APP_STORE_USERNAME\"]"
      password "ENV[\"APP_STORE_PASSWORD\"]"
    end
```

Any ideas what is up?

**Comment by RoboticCheese** [comment]
That's... unusual...

`accessibility/extras` is provided by the  `accessibility_core` gem, which `AXElements` should be pulling in as a dependency when it gets installed just prior to that.

Does the Chef log before that error show it installing the `AXElements` gem and succeeding? Is there anything unusual about the Ruby environment you're running Chef under?

My initial guess would be an issue with Chef getting confused about its Ruby environment--installing gems into one and then trying to load them from another. Do other `chef_gem` resources work properly?

**Comment by RoboticCheese** [comment]
This should be resolved by #31 which will soon be released as 2.0 and get rid of all the accessibility wiring entirely.
